### PR TITLE
Allow text drag and drop in note edit panel

### PR DESCRIPTION
**Pull request by fnkabit**
Feature #1888 
Video demo: https://youtu.be/UHil5b90OWI

### CODE REVIEW

File: src/widgets/qownnotesmarkdowntextedit.cpp [hunk 1]
```diff
@@ -408,6 +408,11 @@ void QOwnNotesMarkdownTextEdit::setMainWindow(MainWindow *mainWindow) {
     this->mainWindow = mainWindow;
 }
 
+bool QOwnNotesMarkdownTextEdit::canInsertFromMimeData(const QMimeData *source) const
+{
+    return (!source->hasUrls());
```
Comment (fnkabit) [inline src/widgets/qownnotesmarkdowntextedit.cpp:413]: text/uri-list mime-types will be handled in MainWindow

**Comment by fnkabit** [comment]
@pbek Tested on Linux only; if you think it should be tested on Windows as well, I could set up an Windows VM tomorrow.

**Comment by pbek** [comment]
Awesome, thank you very much!

> @pbek Tested on Linux only; if you think it should be tested on Windows as well, I could set up an Windows VM tomorrow.

It will not break very important stuff, we can fix that afterwards if there really is a problem. 😁

**Comment by fnkabit** [comment]
That was fast :) Thank you for the review @pbek .
Have a great weekend !

**Comment by pbek** [comment]
Thanks a lot for your contributions! (I'm still working on webpage automation 😁). Have a great weekend yourself!

**Comment by alexanderino** [comment]
I can confirm that drag and drop works perfectly on Windows 8.1 Pro x64. This makes a huge difference to me, so thanks a lot, @fnkabit !

**Pull request merged**